www/src/components/layer-model/image-model/image-content-sections.js

```diff
@@ -0,0 +1,154 @@
+  <LayerContentWrapper
+    key={`content-wrapper${index}`}
+    index={index}
+    displayCodeFullWidth={displayCodeFullWidth}
+  >
+    <ExampleWrapper>
+      <CodeWrapper title="gatsby-config.js" language="javascript">
+        {`plugins: [
+  \`gatsby-transformer-sharp\`,
+  \`gatsby-plugin-sharp\`,
+  {
+    resolve: \`gatsby-source-filesystem\`,
+    options: {
+      path: \`\${__dirname}/src/images\`,
+    },
+  },
+  ...
+]`}
+      </CodeWrapper>
+    </ExampleWrapper>
+    <div>
+      <p>
+        When you set up the <b>Config</b> for your site, you'll want to include{" "}
+        <code>gatsby-transformer-sharp</code> and{" "}
+        <code>gatsby-plugin-sharp</code>.
+      </p>
+      <p>
+        You'll also need to configure <code>gatsby-source-filesystem</code> with
+        the appropriate options. The <code>path</code> should be the path to the
+        directory your image files live in.
+      </p>
+    </div>
+  </LayerContentWrapper>
+)
+
+const QueryLayerContent = ({ index, displayCodeFullWidth }) => (
+  <LayerContentWrapper
+    key={`content-wrapper${index}`}
+    index={index}
+    displayCodeFullWidth={displayCodeFullWidth}
+  >
+    <ExampleWrapper>
+      <CodeWrapper title="src/pages/index.js" language="graphql">
+        {`const query = graphql\`
+  query {
+    file(relativePath: { eq: "images/gatsby-logo.png" }) {
+      childImageSharp {
+        fluid {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+\`
+`}
+      </CodeWrapper>
+    </ExampleWrapper>
+    <div>
+      <p>
+        In order to take advantage of fast image processing in Gatsby, you need
+        to <b>Query</b> your image using GraphQL. Under the hood, this query is
+        processed using <code>gatsby-transformer-sharp</code>.
+      </p>
+      <p>
+        When querying for a specific image you use the <code>relativePath</code>
+        . This path is relative to the path you configured for the{" "}
+        <code>gatsby-source-filesystem</code> plugin and should ultimately point
+        to your image file.
+      </p>
+      <p>
+        Inside the query you'll notice <code>fluid</code> and{" "}
+        <code>GatsbyImageSharpFluid</code> query terms. There are other{` `}{" "}
+        <Link to="/packages/gatsby-image/#two-types-of-responsive-images">
+          types of image processing
+        </Link>{" "}
+        you can prompt with your query which will alter both of those query
+        terms.
+      </p>
+    </div>
+  </LayerContentWrapper>
+)
+
+const DisplayLayerContent = ({ index, displayCodeFullWidth }) => (
+  <LayerContentWrapper
+    key={`content-wrapper${index}`}
+    index={index}
+    displayCodeFullWidth={displayCodeFullWidth}
+  >
+    <ExampleWrapper>
+      <CodeWrapper title="src/pages/index.js" language="jsx">
+        {`import Img from "gatsby-image"
+        export ({ data }) => (
+  <div>
+  <Img fluid={data.file.childImageSharp.fluid} alt="Gatsby logo" />
```

Comment:
```suggestion
export ({ data }) => (
  <div>
    <Img fluid={data.file.childImageSharp.fluid} alt="Gatsby logo" />
```

I think this would help the code example render with a little better formatting 🙂